package.json

```diff
@@ -44,6 +44,8 @@
   "devDependencies": {
     "babel-core": "6.6.0",
     "babel-eslint": "5.0.0",
+    "babel-istanbul": "^0.6.0",
```

Comment:
@kumar303 @muffinresearch sorry, I installed the deps using "npm install --save-dev" and I didn't notice the exact requirements of all the other deps.